src/test/kotlin/io/github/kryszak/gwatlin/api/achievement/AchievementsClientTest.kt

```diff
@@ -6,6 +6,8 @@ import io.kotest.assertions.assertSoftly
 import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.matchers.collections.shouldContainExactly
 import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.collections.shouldMatchEach
```

Comment:
Seems like unused import

Reply:
Good catch. Seems I've grown reliant on linters 🙈

Reply:
Like we all, like we all :smile: